lib/rails_admin/config/actions/base.rb

```diff
@@ -18,6 +18,11 @@ class Base
           []
         end
 
+        # Should the action show a confirmation (using jquery_ujs) before performing this action
+        register_instance_option :confirmation_required do
+          false
+        end
```

Comment:
Changing the name to `:confirmation` and letting users set customized message will be more flexible, IMHO.